nbs/00_export.ipynb

```diff
@@ -4,36 +4,37 @@
    "cell_type": "code",
```

Comment:
<p>There is some weird MarkDown in this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
I think I've upset the ReviewNB parser/renderer again )o:

here's how it looks on github:
![image](https://user-images.githubusercontent.com/11458288/86754130-8b8a0000-c038-11ea-81be-6a72fdf23a72.png)

here's how it looks when I build docs locally:
![image](https://user-images.githubusercontent.com/11458288/86754469-d1df5f00-c038-11ea-9cc9-a79283fc8476.png)

Reply:
In case you're interested, I've created https://github.com/ReviewNB/support/issues/63 to see if they can fix this

Reply:
Thanks for reporting the issue!